English comments and Config from file

tdltdc opened the pull request.

dennisvisschers committed:
solved issues: English comments and Config from file

tdltdc committed:
Do not provide defaults for channel_id and api_key

tdltdc committed:
Do not put part of static API urls in config

These URLs are not subject to change, so there's no need to put them in variables.

dennisvisschers commented on sbsc.py:
```diff
@@ -15,81 +16,91 @@
+conf = Configuration()
+conf.load_json("./config.json")
 
-# Wacht tot systeemklok via NTP is gesynchroniseerd
```
no English comment to replace Dutch comment

tdltdc committed:
Simplify config file handling

We don't need to set the config variables, we only need to read them. So simple reading them in is sufficient

tdltdc committed:
Do not let user set refresh interval of NTP synchronisation and internet connection status

1 second is a reasonable default that most likely noone will object to, so putting it in a variable adds unnecessary complexity. We can always implement it later if there is a need

dennisvisschers commented:
i'm OK with this, if it can be tested before taking into production

dennisvisschers commented:
i probably closed the issues too soon; should have let the PR take care of it

tdltdc committed:
Do not put window timeout in variable

The timeout variable determines the rate at which the warning sign blinks. There's little added value in make it user settable (in fact, it will probably confuse them, because its purpose is not obvious just from reading the code).

tdltdc committed:
Comment polishing

tdltdc committed:
Do not make window resizability user settable, but do so for fullscreen

The window contents are of a fixed and absolute size, so there's not much of a point in resizing the window. But whether it should be fullscreen or not is something that can certainly differ per use case.

tdltdc committed:
Code polishing

tdltdc commented:
> i'm OK with this, if it can be tested before taking into production

I've tested it with the new configuration file. All seems to be working as before (i.e. well ;).

tdltdc committed:
Update README
